backend/tests/test_image_exif.py

```
@@ -0,0 +1,67 @@
+    # 'exif' key should definitely be gone
+    assert 'exif' not in final_img.info
+    # 'test_key' should be gone
+    assert 'test_key' not in final_img.info
+
+def test_save_file_blocking_strips_exif(tmp_path):
+    """
+    Test that save_file_blocking correctly strips metadata.
+    """
+    # Create an image with metadata
+    img = Image.new('RGB', (100, 100), color='blue')
+    img.info['secret'] = 'data'
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "secret"

(S105)